dev-python/ipympl/ipympl-0.9.7.ebuild

```
@@ -0,0 +1,39 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+
+DISTUTILS_USE_PEP517=hatchling
+inherit distutils-r1 pypi
+
+DESCRIPTION="Matplotlib extension for Jupyter"
+HOMEPAGE="https://matplotlib.org/ipympl https://github.com/matplotlib/ipympl https://pypi.org/project/ipympl"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
```

```suggestion
RDEPEND="
```

`DEPEND` is for packages that you link against (in the C sense), it is rarely used in Python packages.

If you really need some of these packages at build time, they should go to `BDEPEND`, but this is unlikely unless upstream is doing something really bad.